workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -0,0 +1,723 @@
+                            "description": "Describes the type of token.",
+                            "type": "array"
+                            "examples"[["fungible","transferable"]]
+                        },
+                        "tokenDesc": {
+                            "$comment": "{\"term\": \"tokenDesc\", \"@id\": \"https://schema.org/description\"}",
+                            "title": "tokenDesc",
+                            "description": "Brief description of the token and its functionality.",
+                            "type": "string",
+                            "examples": ["Protocol Token for the Matic Network"]
+                        },
+                        "standard": {
+                            "$comment": "{\"term\": \"standard\", \"@id\": \"https://schema.org/citation\"}",
+                            "title": "standard",
+                            "description": "Describes which standard the token adheres to.",
+                            "type": "string",
+                            "examples": ["ERC20"]
+                        },
+                        "address": {
+                            "$comment": "{\"term\": \"address\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "address",
+                            "description": "Address of the token smart contract.",
+                            "type": "string",
+                            "examples": ["0x0000000000000000000000000000000000001010"]
+                        },
+                        "addressType": {
+                            "$comment": "{\"term\": \"address\", \"@id\": \"https://schema.org/Intangible\"}",
+                            "title": "addressType",
+                            "description": "AddressType of the token smart contract.",
+                            "type": "string",
+                            "examples": ["MaticNameSpace"]
+                        },
+                        "addressAlg": {
+                            "$comment": "{\"term\": \"addressAlg\", \"@id\": \"https://schema.org/algorithm\"}",
+                            "title": "addressAlg",
+                            "description": "Algorithm used to create the address e.g. CREATE2 or the standard ethereum address construction which is the last 40 characters/20 bytes of the Keccak-256 hash of a secp256k1 public key.",
+                            "type": "string",
+                            "examples": ["CREATE2"]
+                        },
+                        "tokenName": {
+                            "$comment": "{\"term\": \"tokenName\", \"@id\": \"https://schema.org/name\"}",
+                            "title": "tokenName",
+                            "description": "Token name.",
+                            "type": "string",
+                            "examples": ["Matic"]
+                        },
+                        "tokenSymbol": {
```

Comment:
As a general rule of thumb, I believe this specification should be a strict superset of the tokenlist specification. This will make it mostly compatible with existing token list tooling while also giving us the extra functionality we need. One nitpick here is then that the tokenlist spec uses the field `symbol` instead of `tokenSymbol`.

Reply:
reverted